docs/01-get-started/01-server-installation.md

```diff
@@ -20,12 +20,14 @@ Follow the Docker installation instructions found here: [https://docs.docker.com
 
 Download the Cadence docker-compose file:
 
+Clone the repo
 ```bash
-curl -O https://raw.githubusercontent.com/cadence-workflow/cadence/master/docker/docker-compose.yml && curl -O https://raw.githubusercontent.com/cadence-workflow/cadence/master/docker/prometheus/prometheus.yml
+git clone https://github.com/cadence-workflow/Cadence-Docs.git
```

Comment:
The user installing cadence server here which is where our docker-compose files live.